File: backend/internal/models/stand.go
package models

// Participant représente un participant à un événement
type Stand struct {
	ID         uint   `gorm:"primaryKey" json:"id"`
	ActorID    uint   `json:"actor_id" gorm:"not null"`
	KermesseID uint   `json:"kermesse_id" gorm:"not null"`
	Name       string `json:"name"`
	Type       string `json:"type"`
	MaxPoint   int    `json:"max_point" gorm:"default:0"`
}

File: client/lib/core/models/stand.dart
class Stand {
  final int id;
  final int actorId;
  final int kermesseId;
  final String name;
  final String type;
  final int maxPoint;
  final int stock;

  Stand({
    required this.id,
    required this.actorId,
    required this.kermesseId,
    required this.type,
    required this.name,
    required this.maxPoint,
    required this.stock,
  });

  factory Stand.fromJson(Map<String, dynamic> json) {
    return Stand(
      id: json['id'],
      actorId: json['actor_id'],
      kermesseId: json['kermesse_id'],
      name: json['name'],
      type: json['type'],
      maxPoint: json['max_point'],
      stock: json['stock'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'actor_id': actorId,
      'kermesse_id': kermesseId,
      'name': name,
      'type': type,
      'max_point': maxPoint,
      'stock': stock,
    };
  }
}